packages/storybook/.storybook/manager-head.html

```diff
@@ -0,0 +1,87 @@
+<style>
+    body {
+       letter-spacing: 0.03rem;
+    }
+
+    #root > div {
+        margin: -10px 0;
+        width: calc(100vw + 10px);
+        height: calc(100vh + 20px);
+    }
+
+    a[title*="Design System"] {
+        margin-left: 15%;
+        margin-bottom: -5px;
+        padding: 0;
+        max-width: 75%;
+    }
+
+    button[title="Shortcuts"] {
+        display: none;
+    }
+
+    .sidebar-subheading.sidebar-subheading {
+        font-size: 1em;
+        font-weight: bold;
+        letter-spacing: normal;
+        text-transform: none;
+        color: #00175C;
+    }
+
+    .sidebar-item[id*="components-"][data-nodetype="group"]:not([id="ods-components-layout"]) > svg[color='primary'] {
+        display: none;
+    }
+
+    .sidebar-item[id*="components-"][data-nodetype="group"]:not([id*="deprecated"]):not([id*="new"]) > svg {
+        display: initial;
+    }
+
+    .sidebar-item > .badge-deprecated::before {
+      background-color: #C11B1B;
+      color: #FFF;
+    }
+
+    .sidebar-item > .badge-new::before {
+      background-color: #268403;
+      color: #FFF;
+    }
+
+    /*
+     * Avoid padding on APIs iframes
+     */
+    .addon-notes-container #panel-tab-content [class^='css-'] {
+        box-sizing: border-box;
+        margin: 0;
+        max-width: inherit;
+        min-height: 85vh;
+        padding: 0;
+    }
+
+    div[role="main"] > div {
+        box-shadow: none;
+        border-left: solid 1px #008fd4;
+    }
+
+    #release-selector {
+        background: none;
+        border-radius: .3em;
+        border: none;
+        color: #000E9C;
+        display: block;
+        font-weight: 600;
+        margin: .3em auto -.5em auto;
+        text-align: center;
+        transition: all ease-in-out .2s;
+    }
+
+    select#release-selector {
+        cursor: pointer;
+    }
+
+    select#release-selector:hover,
+    select#release-selector:focus {
```

Comment:
is this whole style actually used?
if so please keep only what is currently useful.
Is it possible to move this in the preview css file?